src/app/tests/suites/certification/ci-pics-values

```diff
@@ -1516,48 +1516,20 @@ DRLK.S.A0028.ReadOnly=1
 DRLK.S.A002c.ReadOnly=1
 
 #write attributes
-DRLK.S.A0004.Write=0
-DRLK.S.A0005.Write=0
-DRLK.S.A0006.Write=0
-DRLK.S.A0021.Write=1
-DRLK.S.A0022.Write=0
-DRLK.S.A0023.Write=1
-DRLK.S.A0024.Write=1
-DRLK.S.A0025.Write=1
-DRLK.S.A0028.Write=0
-DRLK.S.A0029.Write=1
-DRLK.S.A002a.Write=1
-DRLK.S.A002b.Write=1
-DRLK.S.A002c.Write=1
-DRLK.S.A0030.Write=1
-DRLK.S.A0031.Write=1
-DRLK.S.A0032.Write=1
-DRLK.S.A0033.Write=1
-DRLK.S.A0035.Write=0
-
-#ReadOnly attributes
-DRLK.S.A0028.ReadOnly=1
-DRLK.S.A002c.ReadOnly=1
+DRLK.S.M.SimulateNotFullyLocked=1
```

Comment:
These PICS references will also need to be changed in Python TC's found in src/python_testing

Reply:
Working on that.

Reply:
@jrhees-cae Updated.